src/assets/scripts/client/aircraft/FlightManagementSystem/RouteModel.js

```diff
@@ -1046,10 +1054,18 @@ export default class RouteModel extends BaseModel {
         }
 
         if (divergentLegModel.isSidLeg) {
+            if (_last(divergentLegModel.waypoints).name === startWaypointName) {
```

Comment:
same here, pull this out to a `const` at the top of the method

Reply:
done